Cargo.toml

```
@@ -24,10 +24,10 @@ default = ["cli"]
 cli = ["dep:clap"]
 
 [dependencies]
-TSPL = "0.0.12"
+TSPL = "0.0.13"
 clap = { version = "4.4.1", features = ["derive"], optional = true }
 highlight_error = "0.1.1"
-hvm = "=2.0.19"
+hvm = "2.0.22"
```

```suggestion
hvm = "=2.0.22"
```
Since HVM doesn't follow semnatic versioning